docs/rbe-platforms.md

```diff
@@ -41,11 +41,41 @@ You can then pass this configuration to BuildBuddy RBE with the following flag:
 
 This assumes you've placed this rule in your root BUILD file. If you place it elsewhere, make sure to update the path accordingly.
 
+### Passing credentials for Docker images
+
+You can pull images from private container registries by adding the following
+flags to your `bazel` command:
+
+```
+--remote_default_exec_properties=container-registry-username=USERNAME
+--remote_default_exec_properties=container-registry-password=ACCESS_TOKEN
+```
+
+For the values of `USERNAME` and `ACCESS_TOKEN`, we recommend generating a
+short lived token using the appropriate utility for your cloud provider.
+
+For GCR (Google Container Registry), the username must be `_dcgcloud_token`
+and a token can be generated with `gcloud auth print-access-token`:
+
+```
+--remote_default_exec_properties=container-registry-username=_dcgcloud_token
+--remote_default_exec_properties=container-registry-password="$(gcloud auth print-access-token)"
+```
+
+For Amazon ECR (Elastic Container Registry), the username must be `AWS`
+and a token can be generated with `aws ecr get-login-password --region REGION`
+(replace `REGION` with the region matching the ECR image URL):
+
+```
+--remote_default_exec_properties=container-registry-username=AWS
+--remote_default_exec_properties=container-registry-password="$(aws ecr get-login-password --region REGION)"
+```
+
```

Comment:
This is great, could you add an example using a custom platform & GCR: https://docs.buildbuddy.io/docs/rbe-platforms#using-a-custom-docker-image

I think this would likely have to use a long lived token (unless you get fancy and generate the platform on the fly - which we can do later in our toolchain).

Reply:
Done